jscomp/others/js_exn.resi

```diff
@@ -31,16 +31,13 @@ type t
 
 type exn += private Error(t)
 
-external asJsExn: exn => option<t> = "?as_js_exn"
+let asJsExn: exn => option<t>
```

Comment:
This means it won't be zero cost anymore?

Reply:
It has runtime in primitive anyway

Reply:
Ok! 👍